src/pages/create-or-edit-question/CreateOrEditQuestion.tsx

```diff
@@ -103,14 +140,32 @@ const CreateOrEditQuestion = () => {
       </AppButton>
       <AppButton
         disabled={!isButtonsVisible}
-        loading={loading}
+        loading={createQuestionLoading || editQuestionLoading}
         type={ButtonTypeEnum.Submit}
       >
         {t('common.save')}
       </AppButton>
     </Box>
   )
 
+  useEffect(() => {
+    if (id) {
+      void handleGetQuestion(id)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [id])
+
+  useEffect(() => {
+    if (question) {
+      handleDataChange<GetQuestion>(question)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
try to remove this disable and set handleDataChange to the dependency array if needed, I think it should work without any troubles

Reply:
You mean to write like this?   
useEffect(() => {
    if (question) {
      handleDataChange<GetQuestion>(question)
    }
  }, [question, handleDataChange])